src/main/resources/public/js/view-process-instance.js

```diff
@@ -1088,31 +1088,107 @@ function isConnectorJob(job) {
 function loadUserTasksOfProcessInstance() {
   const processInstanceKey = getProcessInstanceKey();
 
+  let indexOffset = 1;
   queryUserTasksByProcessInstance(processInstanceKey).done(function (response) {
     let processInstance = response.data.processInstance;
     let userTasks = processInstance.userTasks;
 
     let totalCount = userTasks.totalCount;
+    $("#user-tasks-total-count").text(totalCount);
+
     let nodes = userTasks.nodes;
 
     // first, remove all user task markers on the BPMN
     removeAllUserTaskActionMarkers();
 
-    // TODO (#67): show user tasks in tab
+    $("#user-tasks-of-process-instance-table tbody").empty();
 
-    nodes.forEach((userTask) => {
+    nodes.forEach((userTask, index) => {
       const bpmnElement = userTask.elementInstance.element;
       const elementId = bpmnElement.elementId;
 
       jobKeyToElementIdMapping[userTask.key] = elementId;
 
+      let candidateGroupsFormatted = "-";
+      if (userTask.candidateGroups) {
+        // candidate groups should be a stringified array of strings
+        candidateGroupsFormatted = userTask.candidateGroups
+          .replaceAll("[", "")
+          .replaceAll("]", "")
+          .replaceAll('"', "");
```

Comment:
:wrench: While this works well, a less "hacky" way to do this would be to parse it as JSON array and then use array methods to convert it to a string:
```suggestion
        candidateGroupsFormatted = JSON.parse(userTask.candidateGroups).join(
          ", "
        );
```

Reply:
Looks better. :smile: :+1: